File: leetcode/javascript/isPalindrome.js
/**
 * @param {number} x
 * @return {boolean}
Given an integer x, return true if x is palindrome integer.

An integer is a palindrome when it reads the same backward as forward.
For example, 121 is a palindrome while 123 is not.
 */
var isPalindrome = function(x) {
    const rev = x.toString().split("").reverse().join("")
    if (x == rev) {
        return true
    }
    return false
};

File: leetcode/python/isPalindrome.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        
        Given an integer x, return true if x is palindrome integer.
        An integer is a palindrome when it reads the same backward as forward.
        """
        if x < 0:
            return False
        rev = list(str(x))
        rev.reverse()
        rev = int("".join(rev))
        if x == rev:
            return True
        return False
